workflows/imagery/standardising.yaml

```
@@ -480,32 +480,31 @@ spec:
           ]
 
     - name: create-config
+      retryStrategy:
+        limit: "2"
       inputs:
         parameters:
           - name: location
+            description: Location of the imagery to create config for
       container:
-        image: "ghcr.io/linz/basemaps/cli:{{=sprig.trim(workflow.parameters['version-basemaps-cli'])}}"
-        command: [node, index.cjs]
+        image: ghcr.io/linz/basemaps/cli:{{ workflow.parameters.version-basemaps-cli }}
```

```suggestion
        image: "ghcr.io/linz/basemaps/cli:{{=sprig.trim(workflow.parameters.['version-basemaps-cli'])}}"
```
The trim and [''] are still needed to strip accidental whitespace?

---

Do we need to remove the dot after parameter? 
```suggestion
        image: "ghcr.io/linz/basemaps/cli:{{=sprig.trim(workflow.parameters['version-basemaps-cli'])}}"
```